RNA-Seq/tutorials/de_novo.md

```diff
@@ -0,0 +1,377 @@
+>    > 
+>    > <details>
+>    > <summary>Click to view answers</summary>
+>    > To obtain the up-regulated genes in the G1E state, we filter the previously generated file (with the significant change in transcript expression) with the expression "c3>0" (the log2 fold changes must be greater than 0). We obtain 129  genes (46.7% of the genes with a significant change in gene expression). For the down-regulated genes in the G1E state, we did the inverse and we find 147 transcripts (53.3% of the genes with a significant change in transcript expression)
+>    > </details>
+>    {: .question}
+{: .hands_on}
+
+In addition to the list of genes, `DESeq2` outputs a graphical summary of the results, useful to evaluate the quality of the experiment:
+
+1. Histogram of *p*-values for all tests
+
+    ![](../images/DESeq2_pval.png)
+
+2. [MA plot](https://en.wikipedia.org/wiki/MA_plot): global view of the relationship between the expression change of conditions (log ratios, M), the average expression strength of the genes (average mean, A), and the ability of the algorithm to detect differential gene expression. The genes that passed the significance threshold (adjusted p-value < 0.1) are colored in red.
+
+    ![](../images/DESeq2_MAplot.png)
+
+3. Principal Component Analysis ([PCA](https://en.wikipedia.org/wiki/Principal_component_analysis)) and the first two axes
+
+    ![](../images/DESeq2_PCA.png)
+
+    Each replicate is plotted as an individual data point. This type of plot is useful for visualizing the overall effect of experimental covariates and batch effects.
+
+4. Heatmap of sample-to-sample distance matrix: overview over similarities and dissimilarities between samples
+
+    ![](../images/DESeq2_heatmap.png)
+
+5. Dispersion estimates: gene-wise estimates (black), the fitted values (red), and the final maximum a posteriori estimates used in testing (blue)
+
+    ![](../images/DESeq2_dispersion1.png)
+
+    This dispersion plot is typical, with the final estimates shrunk from the gene-wise estimates towards the fitted estimates. Some gene-wise estimates are flagged as outliers and not shrunk towards the fitted value. The amount of shrinkage can be more or less than seen here, depending on the sample size, the number of coefficients, the row mean and the variability of the gene-wise estimates.
+
+
+For more information about `DESeq2` and its outputs, you can have a look at [`DESeq2` documentation](https://www.bioconductor.org/packages/release/bioc/manuals/DESeq2/man/DESeq2.pdf).
+
+# Visualization
+Now that we have a list of transcript expression levels and their differential expression levels, it is time to visually inspect our transcript structures and the reads they were predicted from. It is a good practice to visually inspect (and present) loci with transcripts of interest. Fortuantely, there is a built-in genome browser in Galaxy, **Trackster**, that make this task simple (and even fun!). 
```

Comment:
It's just a feeling, but don't know about much Galaxy instances that offer a working trackster experience. IGV can be installed by everyone, very easily. I think we can leave this in, but maybe we should just add a sentence, a link to IGV or others (if we have) to just give pointers.

I totally agree that trackster would be the way to go it is actively maintained.